packages/editor/src/components/block-list/style.scss

```
@@ -168,7 +168,7 @@
 		background-color: $blue-medium-highlight;
 	}
 
-	// selection style for multiple blocks
+	// Selection style for multiple blocks.
 	&.is-multi-selected *::selection {
```

```suggestion
	&.is-multi-selected ::selection {
```
I think that asterisk is redundant.

---

Good thought, but I'd prefer not actually making this change in this PR. It is already sort of big surgery and I'd like to minimize the changes as much as possible. Given that code did not change in this PR, only the comment, I'd prefer to look at this separately.